aws/aws.go

```diff
@@ -177,8 +193,23 @@ func NukeAllResources(account *AwsAccountResources, regions []string) error {
 
 		resourcesInRegion := account.Resources[region]
 		for _, resources := range resourcesInRegion.Resources {
-			if err := resources.Nuke(session); err != nil {
-				return errors.WithStackTrace(err)
+			length := len(resources.ResourceIdentifiers())
+			if length >= MaxRequestSize {
```

Comment:
Actually, it doesn't seem like you should even do this check. Instead, always batch things. If there are fewer items than the `MaxBatchSize`, it will all end up in one batch anyway.

Reply:
I'll remove this and up the batch size of all resources to 500